Wikipedia/Images/WMFImageDownload.swift

```diff
@@ -12,6 +12,7 @@ public enum ImageOrigin: String {
     case Network = "Network"
     case Disk = "Disk"
     case Memory = "Memory"
+    case None = "None"
```

Comment:
Isn't case none nonsensical?
Are you passing an imageOrigin even if no image is returned? Shouldn't it just be an error with no origin?